elm-package.json

```diff
@@ -10,8 +10,8 @@
         "Data.Decimal"
     ],
     "dependencies": {
-        "elm-lang/core": "3.0.0 <= v < 4.0.0",
-        "javcasas/elm-integer": "1.0.0 <= v < 2.0.0"
+        "elm-lang/core": "4.0.1 <= v < 5.0.0",
+        "javcasas/elm-integer": "2.0.0 <= v < 3.0.0"
```

Comment:
@javcasas I saw that `elm-integer` has been updated to version [2.0.1](http://package.elm-lang.org/packages/javcasas/elm-integer/2.0.1). In this case, you should update minimum dependency version there too after merging this pull request.